js/apps/admin-ui/src/PageHeader.tsx

```diff
@@ -181,20 +197,21 @@ export const Header = () => {
     : toDashboard({ realm });
 
   return (
-    <PageHeader
-      showNavToggle
-      logo={
-        <Link to={logoUrl}>
-          <Brand
-            src={environment.resourceUrl + logo}
-            id="masthead-logo"
-            alt="Logo"
-            className="keycloak__pageheader_brand"
-          />
-        </Link>
-      }
-      logoComponent="div"
-      headerTools={headerTools()}
-    />
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton variant="plain" aria-label={t("navigation")}>
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadBrand href={logoUrl as string}>
+        <img
+          src={environment.resourceUrl + logo}
+          id="masthead-logo"
+          alt="Logo"
```

Comment:
This `alt` text will have to be localized. Also this will need an `aria-label`, or a `role` to indicate the image is not not to be read by a screen reader.